### PR TITLE
Changed video-timeline workflow to use user_id instead of username.

### PR DESCRIPTION
**Pull request by HassanJaveed84**
Validation pending on history generation(in progress).

### CODE REVIEW

File: edx/analytics/tasks/insights/video.py [hunk 1]
```diff
@@ -183,13 +183,21 @@ def mapper(self, line):
...
-            log.error("Video event without username: %s", event)
-            # Slow: self.incr_counter(self.counter_category_name, 'Discard Video Missing Something', 1)
-            # Slow: self.incr_counter(self.counter_category_name, 'Discard Video Missing username', 1)
+        # # Strip username to remove trailing newlines that mess up Luigi.
+        # username = event.get('username', '').strip()
+        # if not username:
+        #     log.error("Video event without username: %s", event)
+        #     # Slow: self.incr_counter(self.counter_category_name, 'Discard Video Missing Something', 1)
+        #     # Slow: self.incr_counter(self.counter_category_name, 'Discard Video Missing username', 1)
+        #     return
```
Comment (brianhw) [inline edx/analytics/tasks/insights/video.py]: This should just be removed, I think.

**Review by brianhw — COMMENTED** [review]
Did we want to change the names of the history directories?   Instead of "user_video_viewing" and "user_video_viewing_by_date", maybe something like "userid_video_viewing"?   That way one could run this in parallel and compare, rather than having to overwrite.   And also so that when people upgrade, they don't get lots of errors from mixing the old with new.   Or "video_viewing_by_user", or just "video_viewing"?

**Comment by HassanJaveed84** [comment]
I did change those, but in a separate branch(which is what I'm using for history generation). I thought after we're done validating, we could keep the original names of the directories. But your point about other people upgrading is very valid. I'll change them here as well.

**Review by brianhw — APPROVED** [review]
LGTM.   👍

**Pull request merged**